spec/models/question_answer_comment_spec.rb

```diff
@@ -40,6 +40,29 @@
         I18n.t("qa.comment.errors.limit_exceeded", limit: SiteSetting.qa_comment_limit_per_post)
       )
     end
+
+    it 'does not allow comment to be created when raw does not meet min_post_length site setting' do
+      SiteSetting.min_post_length = 5
```

Comment:
Since we're adding a plugin-specific site setting for `max_length`, maybe we should add one for `min_length` as well. Would be more consistent, plus QA comments are different data types from posts, I can see sites wanting different values there from core.

Reply:
I thought about this but I didn't feel like the plugin should deviate from core in terms of what the minimum length should be. I deviating from core in terms of the max length for comments is because comments are primarily used to seek clarifications on the answer and not to be used for discussions.